content/pt-br/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md

```diff
@@ -0,0 +1,109 @@
+---
+reviewers:
+- danwent
+- aanm
+title: Utilize o Cilium para NetworkPolicy
+content_type: task
+weight: 30
+---
+
+<!-- overview -->
+Essa página mostra como utilizar o Cilium para NetworkPolicy.
+
+Para saber mais sobre o Cilium, leia o artigo [Introduction to Cilium](https://docs.cilium.io/en/stable/overview/intro).
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+## Fazendo o Deploy do Cilium no Minikube para Testes Básicos
+
+Para familiarizar-se com o Cilium você poderá seguir o guia [Cilium Kubernetes Getting Started Guide](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/) e realizar uma instalação básica do Cilium através de um DaemonSet no minikube.
```

Comment:
"o guia [Cilium Kubernetes Getting Started Guide]" ->  could be "o [Guia  de Primeiros Passos do Cillium no Kubernetes]"

Reply:
ok! I thought of leaving the title in English because the link redirects to an article written in English. Do you think this should be pointed out? Or do we leave as you suggested?